polkadot/node/network/availability-recovery/Cargo.toml

```diff
@@ -51,3 +51,4 @@ required-features = ["subsystem-benchmarks"]
 
 [features]
 subsystem-benchmarks = []
+subsystem-benchmarks-helpers = []
```

Comment:
`subsystem-benchmarks` feature seems to no longer be used at all.

Reply:
It's in use. We require that feature to run regression tests. That way we don't mix normal and regression tests.